box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseActivity.java

```diff
@@ -226,29 +204,51 @@ private void clearSearch() {
     }
 
     @Override
-    public void onMoreResultsRequested(BoxRequestsSearch.Search searchRequest) {
-        clearSearch();
+    public void onItemClick(BoxItem item) {
+        // Notify fragment about item click
+        BoxBrowseFragment browseFrag = getTopBrowseFragment();
```

Comment:
After thinking about it, I don't like this round trip behavior where the fragment is calling the activity which is then calling the fragment. I believe the appropriate behavior is for the fragment to already know the item click.